content/00.zksync-era/40.tooling/20.hardhat/30.plugins/35.hardhat-zksync.md

```diff
@@ -49,34 +57,123 @@ bun add @matterlabs/hardhat-zksync --dev
 
 ::
 
-### Usage
+Import the package in your `hardhat.config.ts` file:
+
+```typescript
+import "@matterlabs/hardhat-zksync";
+```
+
+## Configuration
 
-After installing it, add the plugin to your Hardhat config:
+The plugin serves as a bundle for other plugins and doesn't require specific configuration itself.
+However, you can configure the individual plugins it includes:
 
-```javascript
+| Plugin | Configuration Options |
+|--------|----------------------|
+| hardhat-zksync-solc | See [hardhat-zksync-solc](/zksync-era/tooling/hardhat/plugins/hardhat-zksync-solc) |
+| hardhat-zksync-deploy | See [hardhat-zksync-deploy](/zksync-era/tooling/hardhat/plugins/hardhat-zksync-deploy) |
+| hardhat-zksync-upgradable | See [hardhat-zksync-upgradable](/zksync-era/tooling/hardhat/plugins/hardhat-zksync-upgradable) |
+| hardhat-zksync-verify | See [hardhat-zksync-verify](/zksync-era/tooling/hardhat/plugins/hardhat-zksync-verify) |
+| hardhat-zksync-node | See [hardhat-zksync-node](/zksync-era/tooling/hardhat/plugins/hardhat-zksync-node) |
+| hardhat-zksync-ethers | See [hardhat-zksync-ethers](/zksync-era/tooling/hardhat/plugins/hardhat-zksync-ethers) |
+
+### Example Configuration
+
+```typescript
 import "@matterlabs/hardhat-zksync";
+import { HardhatUserConfig } from "hardhat/config";
+
+const config: HardhatUserConfig = {
```

Comment:
It would be great to have an example or reference will all the options currently available.

We've made [a versioned reference](https://matter-labs.github.io/era-compiler-solidity/latest/03-standard-json.html#input-json) for standard JSON input that is easy to follow and synchronize with.

Some options are still unsupported by Hardhat, so I propose finalizing this update first, and then support the newly added features.